Core1Platform/AdministratingCore1/00_AdministratingCore1.md

```diff
@@ -0,0 +1,39 @@
+# Administrating the Core1 Platform
+<!--- Links getestet-->
+In this part of the *Actindo Core1 Platform* documentation, you will find the following chapters:
+
+- [Main account and sandboxes](./01_MainAccountSandbox.md)   
+
+- [Manage users](./02_ManageUsers.md)
+    - [Create user](./02_ManageUsers.md#create-user)
+    - [Assign user to user group](./02_ManageUsers.md#assign-user-to-user-group)
+    - [Edit user](./02_ManageUsers.md#edit-user)
+    - [Remove user from a group](./02_ManageUsers.md#remove-user-from-a-group)
+    - [Invite user to sandbox](./02_ManageUsers.md#invite-user-to-sandbox)
+    - [Remove user in sandbox](./02_ManageUsers.md#remove-user-in-sandbox) 
+    - [Delete user](./02_ManageUsers.md#delete-user) 
+
```

Comment:
Ist es sinnvoll, die Procedures zu gruppieren, z.B.:
- Create user
- Edit user
   - Assign a user to a user group
   - Remover a user from a user group
   - Invite a user to a sandbox
   - Remove a user from a sandbox
- Delete a user

Reply:
Es handelt sich um unterschiedliche Aktivitäten, die weniger mit Edit user zu tun haben